site2/docs/reference-cli-tools.md

```diff
@@ -6,1066 +6,21 @@ sidebar_label: "Pulsar CLI tools"
 
 Pulsar offers several command-line tools that you can use for managing Pulsar installations, performance testing, using command-line producers and consumers, and more.
 
-All Pulsar command-line tools can be run from the `bin` directory of your [installed Pulsar package](getting-started-standalone.md). The following tools are currently documented:
+* [`pulsar-admin`](https://pulsar.apache.org/reference/#/latest/pulsar-admin/)
+* [`pulsar`](https://pulsar.apache.org/reference/#/latest/pulsar/)
+* [`pulsar-client`](https://pulsar.apache.org/reference/#/latest/pulsar-client/)
+* [`pulsar-perf`](https://pulsar.apache.org/reference/#/latest/pulsar-perf/)
```

Comment:
1. We'd better use `/reference` than `https://pulsar.apache.org/reference` here.
2. We should find a way to generate version part (`latest`) so that when the next version become a stable version, it doesn't direct users to the latest reference pages.

Reply:
Perhaps we can use `@pulsar:version@` magic var but I think it cannot be resolved as `latest`. Maybe `@pulsar:docversion`? But there will be an issue between `latest` and `next`. We should unify names here.

Reply:
We can change `latest` to `next`, but may not be able to use `/reference` as it may be parsed by Docusaurus as a markdown docs rather than an external link to another site.

Reply:
@SignorMercurio Let's do the first part.

I'm thinking of that you can directly move the `docsify` folder under `website-next/static/` and we can preview the references pages also. You can give it a try. And perhaps several build logics should be modified.